tests/getex_test.go

```diff
@@ -0,0 +1,119 @@
+package tests
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetEx(t *testing.T) {
```

Comment:
Add a test case with invalid syntax (multiple expiry options specified)

Reply:
Hi @JyotinderSingh, added test cases and handled the case with multiple expiry options